### PR TITLE
Completes all the PHP API Adapter code for localws <-> 2015 API

### PR DESCRIPTION
**Pull request by jwisener**
This completes the php api adapter subset of functions to access and use the new 2015 API. 

The README_2015_API.md that has been added describes quite bit of what has and has not been implemented.

I don't know if we need to create a new repo or merge this into the existing repo. 

Testing
=====================
I ran all implemented methods calls against localws and an upgraded (localws) database with engine 2015 and compared the output to check to make sure the structure was the same.

### CODE REVIEW

**Comment by nathaninlatin** [comment]
@tseabrooks we need to figure out how we'll handle the repos on this. Do we:

1. Tag the API version 0 library and update the links on our sites to point to that repo (until we're ready to go live with this)
2. Put this in an entirely separate repo.
3. Put this in a different branch of the SCORMCloud_PHPLibrary, where it will live until we make it master.

I vote for 3…probably easiest on everyone.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,35 @@
+### PhpStorm ###
```
Comment (nathaninlatin) [inline .gitignore]: So I don't think we want to add .gitignore to this repo. End-users will be dropping this into their php webserver directories and .gitignore will be an unnecessary file for them.

It's also possible you didn't intend to add this file, so: heads-up. (You may want to add gitignore to your gitignore.)

**Comment by tseabrooks** [comment]
@nathaninlatin @jwisener Make a new internal repo for this.

File: .gitignore (hunk 1 above)
Comment (jwisener) [inline .gitignore]: Thanks Nathan. I can definitely remove the .gitignore I was using php storm for development, so I didn't want the .idea folder or files showing up.

**Pull request closed without merging**